lightyear/src/server/networking.rs

```diff
@@ -170,6 +189,12 @@ pub(crate) fn receive_packets(
         }
     }
 
+    for (server_idx, error) in client_errors {
```

Comment:
We do this here and not above because of borrow checker issues?

Reply:
Yeah, borrow checker issues abound trying to collect the things I need to disconnect. Maybe it's better with ServerCommands for disconnects, I'll see if I can revert some of the rearrangement when I do that